main.js

```diff
@@ -0,0 +1,99 @@
+// let gc = document.querySelector('#gc');
+
+// gc.addEventListener('click', (e) =>
+// console.log(e.target.id +' was clicked'))
+
+let items = document.querySelectorAll('.card')
+
+let hasFlippedCard = false;
+let lockBoard = false;
+let firstCard, secondCard;
+let matched = ''
+
+
+function flipCard() {
+    if (lockBoard) return;
+    if (this === firstCard) return;
+    this.classList.add('flip');
+
+if (!hasFlippedCard) {
+    hasFlippedCard = true;
+    firstCard = this;
+    return;
+}
+secondCard=this;
+
+
+checkForMatch()
+}
+function checkForMatch() {
+    let isMatch = firstCard.dataset.framework === secondCard.dataset.framework;
+    isMatch ? disableCards() : unflipCards();
+}
+
+function disableCards() {
+    firstCard.removeEventListener('click', flipCard);
+    secondCard.removeEventListener('click', flipCard);
+    resetBoard();
+    console.log (matched += '1')
+    alert('Correct!')
```

Comment:
Can you make it so that the second matched card is flipped before you run the alert?